src/main/java/com/microsoft/sqlserver/jdbc/SQLServerResource.java

```diff
@@ -502,6 +503,7 @@ static String getResource(String key) {
         {"R_crClientUnrecoverable","The connection is broken and recovery is not possible. The connection is marked by the client driver as unrecoverable. No attempt was made to restore the connection."},
         {"R_crClientSSLStateNotRecoverable", "The server did not preserve SSL encryption during a recovery attempt, connection recovery is not possible."},
         {"R_crCommandCannotTimeOut", "Request failed to time out and SQLServerConnection does not exist"},
+        {"R_InvalidIPAddressPreference", "Invalid option for connection string option IPAddressPreference: {0}"},
```

Comment:
sentence is awkward...can you check the other strings for consistency. Usually they are something like invalid <blah> specified: {0}

Reply:
Changed to be similar to other outputs.